.travis.yml

```diff
@@ -14,4 +14,3 @@ script:
   - yarn lint
   - yarn pretty-check
   - yarn test
-  - npx mocha test/printingTools.js
```

Comment:
How is this test called, once this line is removed?

Reply:
I don't understand the question here. Is it called or not? Why are we removing it?